include/evmc/evmc.h

```diff
@@ -494,39 +485,24 @@ typedef enum evmc_storage_status (*evmc_set_storage_fn)(struct evmc_context* con
  *
  * This callback function is used by a VM to query the balance of the given address.
  *
- * @param[out] result   The pointer to the place where to put the result balance.
- *                      The pointed memory is only modified when the function returns true.
- *                      The pointer MUST NOT be null.
- * @param      context  The pointer to the Host execution context.
- * @param      address  The address of the account.
- * @return              If the account exists its balance is put at the location
- *                      pointed by @p result and true is returned.
- *                      If the account does not exist false is returned without
- *                      modifying the memory pointed by @p result.
+ * @param context  The pointer to the Host execution context.
+ * @param address  The address of the account.
+ * @return         The balance of the given storage key or 0 if the account does not exist.
```

Comment:
The balance of the given account

Reply:
Still a typo here

Reply:
Yes. Funny. It looks like this typo was introduced in some other commit than I thought.